scripts/ci/ipxe-run.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+if [[ $# -ne 2 ]]; then
+    echo "Usage: $0 <dts-base-image-path> <bzImage-path>"
+    exit 1
+fi
+
+DTS_IMAGE_PATH=$1
+DTS_IMAGE_FILENAME=$( basename $DTS_IMAGE_PATH )
+BZ_IMAGE_PATH=$2
+BZ_IMAGE_FILENAME=$( basename $BZ_IMAGE_PATH )
+IPXE_PATH=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)/../../ipxe
+
+mkdir -p $IPXE_PATH
+ln -sf $DTS_IMAGE_PATH $IPXE_PATH/$DTS_IMAGE_FILENAME
+ln -sf $BZ_IMAGE_PATH $IPXE_PATH/$BZ_IMAGE_FILENAME
+
+echo -e "\n
+#!ipxe\n
+imgfetch --name file_kernel $BZ_IMAGE_FILENAME\n
+imgfetch --name file_initrd $DTS_IMAGE_FILENAME\n
+kernel file_kernel root=/dev/nfs initrd=file_initrd\n
+boot" > $IPXE_PATH/dts.ipxe
```

Comment:
```suggestion
echo "
#!ipxe
imgfetch --name file_kernel $BZ_IMAGE_FILENAME
imgfetch --name file_initrd $DTS_IMAGE_FILENAME
kernel file_kernel root=/dev/nfs initrd=file_initrd
boot" > "$IPXE_PATH/dts.ipxe"
```

or

```suggestion
cat <<EOF > "$IPXE_PATH/dts.ipxe"
#!ipxe
imgfetch --name file_kernel $BZ_IMAGE_FILENAME
imgfetch --name file_initrd $DTS_IMAGE_FILENAME
kernel file_kernel root=/dev/nfs initrd=file_initrd
boot
EOF
```

Not sure how imgfetch deals with arguments with spaces so I guess we can leave it and hope filename won't have space

Reply:
Second one is probably better as first one creates file that has first line empty:

```diff
diff dts*
1,2c1,2
<
< #imgfetch --name file_kernel test1
---
> #!ipxe
> imgfetch --name file_kernel test1
```

This first line results in:

```
chain http://127.0.0.1:4321/dts.ipxe
http://127.0.0.1:4321/dts.ipxe... ok
Could not boot: Exec format error (https://ipxe.org/2e008081)
```

Using second format:

```
iPXE> chain http://127.0.0.1:4321/dts.ipxe
http://127.0.0.1:4321/dts.ipxe... ok
README.md... ok
LICENSE.md... ok
```

Reply:
Second one is probably better as first one creates file that has first line empty:

```diff
diff dts*
1,2c1,2
<
< #imgfetch --name file_kernel test1
---
> #!ipxe
> imgfetch --name file_kernel test1
```

This first line results in:

```
chain http://127.0.0.1:4321/dts.ipxe
http://127.0.0.1:4321/dts.ipxe... ok
Could not boot: Exec format error (https://ipxe.org/2e008081)
```

Using second format:

```
iPXE> chain http://127.0.0.1:4321/dts.ipxe
http://127.0.0.1:4321/dts.ipxe... ok
README.md... ok
LICENSE.md... ok
```

Not only that but you can use second format in interactive terminal, first one doesn't work for me (at least copy paste doesn't work)

Reply:
Okay, changed to the second format in: 8d153954c80a3bae8e8b6e32e06bbbb6c02b702b